src/applications/disability-benefits/2346/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Form-2346",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "mdt-introduction",
+  "rootUrl": "/mdt/2346",
+  "production": false
+}
```

Comment:
Make sure to also include this block ([reference](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/documentation/src/pages/getting-started/common-tasks/new-application.mdx)):

```js
  "template": {
    "vagovprod": false
  }
```

This prevents the page from appearing in production. I think it also keeps the page from showing up in the search results.

Reply:
Just added it. Thanks